ruby/examples/app/controllers/user_registration_controller.rb

```diff
@@ -0,0 +1,31 @@
+class UserRegistrationController < ApplicationController
+  def new
+    @user = UserRegistration.new
+  end
+
+  def create
+    request = RegisterUser::Request.new(*user_params)
+    response = RegisterUser.new.register(request)
```

Comment:
I think there's a lack of dependency container libraries in ruby because the idea isn't that necessary. I'd like to grow a code base trying to avoid them and the boilerplate and see what challenges that presents. I imagine we'd find a more ruby way of solving it.